external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpoutMessageId.java

```diff
@@ -21,7 +21,9 @@
 import org.apache.kafka.clients.consumer.ConsumerRecord;
 import org.apache.kafka.common.TopicPartition;
 
-public class KafkaSpoutMessageId {
+import java.io.Serializable;
+
+public class KafkaSpoutMessageId implements Serializable{
     private final transient TopicPartition topicPart;
```

Comment:
I think they probably shouldn't be. The eventlogger will be more useful if it can log topicpartition and offset.